tests/test_remote_write_text_atomic.py

```
@@ -0,0 +1,176 @@
+
+    def crash_on_target_rename(self: Path, target_path):
+        if Path(target_path) == target:
+            raise OSError("simulated crash mid-rename")
+        return real_replace(self, target_path)
+
+    monkeypatch.setattr(Path, "write_text", crash_on_direct_target)
+    monkeypatch.setattr(Path, "replace", crash_on_target_rename)
+    try:
+        jobs_module._write_text(target, "new-content\n")
+    except Exception:
+        pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)